test/ci.sh

```diff
@@ -9,14 +9,6 @@ mac_steps() {
 }
 
 linux_steps() {
-  if ! available autopep8; then
```

Comment:
Changing to pip, that way we can lock in on a common version for Fedora, RHEL, Ubuntu, macOS, etc.

These are only build tools anyway...

Reply:
Not sure if this is going to be allowed on RHEL and Fedora builds, but fine.

Reply:
Yeah on real builds, like rpm packages, maybe won't don't need to run codespell or autopep8 at all, or we can use the packages....